spec/message.abnf

```diff
@@ -1,19 +1,15 @@
 message = [s] *(declaration [s]) body [s]
 
-declaration = let s variable [s] "=" [s] "{" [s] expression [s] "}"
+declaration = let s variable [s] "=" [s] placeholder
 body = pattern
      / (selectors 1*([s] variant))
 
 pattern = "{" *(text / placeholder) "}"
-selectors = match 1*([s] selector)
-selector = "{" [s] expression [s] "}"
+selectors = match 1*([s] placeholder)
 variant = when 1*(s key) [s] pattern
 key = nmtoken / literal / "*"
 
 placeholder = "{" [s] expression [s] "}"
```

Comment:
Do we need `placeholder` at all now?

Reply:
If we didn't have `placeholder`, the `"{" [s] ... [s] "}"` would need to be repeated in three different places. On the other hand, `expression` could be dropped by inlining its contents into `placeholder`, which is the only place where it's now used.

Reply:
I meant dropping `placeholder` and moving the braces to `expression`. I prefer “expression” because I wouldn’t call the RHS in a `let` declaration a “placeholder”.

Reply:
Would you prefer including that change in this PR, or as a separate follow-up?

Reply:
Done.